C++/Kadane-Algorithm/Kadane.cpp

```diff
@@ -0,0 +1,60 @@
+#include <bits/stdc++.h>
```

Comment:
Please put this program in **Sub-Array** folder

Reply:
ok